tests/settings.toml.example

```diff
@@ -1,4 +1,4 @@
 [cli]
-base_url = "http://localhost:8080"
+base_url = "http://localhost"
```

Comment:
Accidental change? The CI uses port 8080 I'm pretty sure.

Reply:
Right. And this file is simply copied to satisfy the CI.

Reply:
EDEV_IS_IDIOT raised.  Fixing.